cypress/integration/shared.js

```diff
@@ -158,6 +158,9 @@ export const downloadFileWithChrome = (downloadButton, resultantFilename) => {
   cy.exec(`[ -f ${resultantFilename} ] && rm -rf ${resultantFilename}`, {
     failOnNonZeroExit: false,
   });
+  cy.wait(2 * 60 * 1000);
+  cy.get('.pf-c-modal-box__footer > .pf-m-primary').click();
+  cy.get(downloadButton, { timeout: 2 * 60 * 1000 }).contains('Download Discovery ISO');
```

Comment:
I think this is too specific for this generic function. From the name, the function can be used to handle other file downloads too.

Reply:
In the end I've removed the lines from there and added sleep to the parent caller file. 
There is some bug in there where while being called via Cypress, 2 images are created at the same time.